java/ql/src/semmle/code/java/frameworks/jackson/JacksonSerializability.qll

```diff
@@ -50,8 +51,17 @@ library class JacksonWriteValueMethod extends Method, TaintPreservingCallable {
   }
 }
 
+private class JacksonReadValueMethod extends Method, TaintPreservingCallable {
+  JacksonReadValueMethod() {
+    getDeclaringType().hasQualifiedName("com.fasterxml.jackson.databind", "ObjectReader") and
+    hasName(["readValue", "readValues"])
+  }
```

Comment:
I think [`ObjectMapper.readValue`](http://fasterxml.github.io/jackson-databind/javadoc/2.12/com/fasterxml/jackson/databind/ObjectMapper.html#readValue-java.lang.String-com.fasterxml.jackson.core.type.TypeReference-) should be considered too
```suggestion
  JacksonReadValueMethod() {
    (
      getDeclaringType().hasQualifiedName("com.fasterxml.jackson.databind", "ObjectReader") or
      getDeclaringType().hasQualifiedName("com.fasterxml.jackson.databind", "ObjectMapper")
    ) and
    hasName(["readValue", "readValues"])
  }

Reply:
I can get behind that! 😄